packages/calcite-components/src/components/popover/popover.scss

```diff
@@ -1,10 +1,17 @@
 /**
- * CSS Custom Properties
- *
- * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-popover-z-index: Sets the z-index value for the component.
- */
+* CSS Custom Properties
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-popover-background-color: Specifies the background color of the component.
+* @prop --calcite-popover-border-color: The border color of the component.
+* @prop --calcite-popover-border-inline-end-color: The border color of the component.
+* @prop --calcite-popover-border-start-end-radius: The start end border radius of the component.
+* @prop --calcite-popover-border-end-end-radius: The end end border radius of the component.
+* @prop --calcite-popover-font-size: The font size of `calcite-option`s in the component.
```

Comment:
I think this could be set with pre existing global vars, not sure if we need the font size per component. Cc @alisonailea to confirm / disconfirm.